pkg/network/config/config.go

```diff
@@ -312,6 +315,8 @@ func New() *Config {
 
 		ProtocolClassificationEnabled: cfg.GetBool(join(netNS, "enable_protocol_classification")),
 
+		RingbuffersEnabled: cfg.GetBool(join(netNS, "enable_ringbuffers")),
```

Comment:
What do you think about making this `true` by default?

Reply:
I think this makes sense now that this has been tested a bit more. I will run another HTTP test with this latest image and post those results on the PR as well